tests/integration/data/on-and-to.on-amd64-to-s390x.yaml

```diff
@@ -0,0 +1,8 @@
+plugin: nil
+source-branch: branch2
```

Comment:
Also surprised by this. The "on/to" behaviour is very counterintuitive.

Reply:
Agreed, I think I have a few bug reports to create in craft-grammar.